### PR TITLE
Refactor room list & clean up some UI. closes #33

### LINKED ISSUE #33: Refactor room list, use state params and state resolve
Gijs en Jan-Willem

### PR DESCRIPTION
**Pull request by GijsWeterings**

### CODE REVIEW

**Comment by Azkle** [comment]
The moods bored and relaxing are  in the frontend, but not in the backend.
The error I get:     org.jboss.resteasy.spi.UnhandledException: java.lang.IllegalArgumentException: No enum constant me.moodcat.mood.Mood.RELAXING

The same for bored.

We should fix this.

For the rest it works fine.

File: app/scripts/controllers/moodCatAudioRating.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+angular.module('moodCatApp')
+  .directive("myDirective", function(){
```
Comment (JWGmeligMeyling) [inline app/scripts/controllers/moodCatAudioRating.js]: Can we rename myDirective?

**Comment by JWGmeligMeyling** [comment]
Looks good, I will build up on this for #34.

**Comment by Azkle** [comment]
LGTM

**Comment by JWGmeligMeyling** [comment]
Merging now.

**Pull request merged**